lib/fetch_blocks_list.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const csv = require('csv-parser');
+const { logger } = require('./logger');
+const { EXPORT_BLOCKS_LIST_MAX_INTERVAL } = require('./constants');
+
+const configPath = process.env.CONFIG_PATH;
+
+/**
+ * @returns {Promise} Promise, resolves to the csv data in Array format
+ */
+const getConfigMapList = async () => {
+  const csvData = [];
+  const readStream = fs.createReadStream(configPath);
+  return new Promise((resolve, reject) => {
+    readStream
+      .pipe(csv())
+      .on('data', (data) => {
+        const blockNumber = Object.values(data)[0];
+        csvData.push(blockNumber);
+      })
+      .on('end', () => {
+        logger.info('Processed data from config file');
+        resolve(csvData);
+      })
+      .on('error', (err) => { console.log(err); reject(err); });
```

Comment:
are we sure we want to log it this place? I mean that once again I expect that if we throw exception then it will be logged at the place you catch it. in this case we'll end up logging it twice

Reply:
Yes, I've missed removing it